<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700;&quot;&gt;串口助手&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>72</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;串口选择：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="mousecombobox" name="Port_comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="accept_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>451</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="send_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BaudRate_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Stop_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Data_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>奇偶校验</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Check_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>串口操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="choice_pushButton">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="time_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="time_checkBox">
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="hex_accept_checkBox">
      <property name="text">
       <string>HEX接收</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_send_pushButton">
      <property name="text">
       <string>清除发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="hex_send_checkBox">
      <property name="text">
       <string>HEX发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_accept_pushButton">
      <property name="text">
       <string>清除接收</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mousecombobox</class>
   <extends>QComboBox</extends>
   <header>mousecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
